### ISSUE TITLE
There is a small wrong mistake in the pytorch official web file 

### ISSUE THREAD

**Issue by dohwan2** [pytorch/pytorch#5732]
When submitting a bug report, please include the following information (where relevant):
- OS: Windows7
- PyTorch version: 0.3.0
- How you installed PyTorch (conda, pip, source): conda
- Python version: 3.5.4
- CUDA/cuDNN version: None
- GPU models and configuration: None
- GCC version (if compiling from source): 

In addition, including the following information will also be very helpful for us to diagnose the problem:
- http://pytorch.org/docs/0.3.1/nn.html#bceloss
- There is no 'reduce' parameter in BCE Loss Function.
  But, the third line says " **If reduce is True**, then ℓ(x,y)={mean(L),sum(L), bla-bla". 
- I think it needs to be deleted.("**If reduce is True**")

**Comment by apaszke** [comment]
0.3.1 docs are frozen but it’s already been fixed in master, so should be ok starting from the next release
